provider/kubernetes/kubernetes_test.go

```diff
@@ -1204,6 +1213,28 @@ func TestIngressClassAnnotation(t *testing.T) {
 				),
 			),
 		},
+		{
+			desc:     "Provided IngressClass annotation",
+			provider: Provider{IngressClass: "custom"},
+			expected: buildConfiguration(
+				backends(
+					backend("herp/derp",
+						servers(
+							server("http://example.com", weight(1)),
```

Comment:
could you update this to be an IP/Port combo instead of a URL?
Currently that is only implemented as a URL when its an externalName service IIRC